relay-general/src/pii/builtin.rs

```diff
@@ -779,6 +832,217 @@ mod tests {
+            ];
+        );
+        assert_custom_rulespec!(
+            rule = "@iban:replace";
+            input = "some iban: DE89370400440532013000!";
+            output = "some iban: [Filtered]!";
+            remarks = vec![
+                Remark::with_range(RemarkType::Substituted, "0", (11, 21)),
+            ];
+        );
+        assert_text_rule!(
+            rule = "@iban:hash";
+            input = "some iban: DE89370400440532013000!";
+            output = "some iban: 8A1248B6F40D38FBC59ADE6AD0DF69C7BB9C936A!";
+            remarks = vec![
+                Remark::with_range(RemarkType::Pseudonymized, "@iban:hash", (11, 51)),
+            ];
+        );
+        assert_custom_rulespec!(
+            rule = "@iban:hash";
+            input = "some iban: DE89370400440532013000!";
+            output = "some iban: 8A1248B6F40D38FBC59ADE6AD0DF69C7BB9C936A!";
+            remarks = vec![
+                Remark::with_range(RemarkType::Pseudonymized, "0", (11, 51)),
+            ];
+        );
+    }
+
+    #[test]
+    fn test_invalid_iban_codes() {
+        let mut valid_norwegian_iban = "NO9386011117945".to_string();
+
+        assert_text_rule!(
+            rule = "@iban";
+            input = &format!("some iban: {}!", valid_norwegian_iban);
+            output = "some iban: [iban]!";
+            remarks = vec![Remark::with_range(RemarkType::Substituted, "@iban", (11, 17))];
+        );
+
+        let invalid_norwegian_iban = {
+            valid_norwegian_iban.pop();
+            valid_norwegian_iban
+        };
+
+        // Since norway has the shortest iban, it won't recognize an iban shorter than that.
+        assert_rule_not_applied!(
+            rule = "@iban";
+            input = &format!("some iban: {}!", invalid_norwegian_iban);
+        );
+
+        let mut valid_russian_iban = "RU0204452560040702810412345678901".to_string();
+
+        assert_text_rule!(
+            rule = "@iban";
+            input = &format!("some iban: {}!", valid_russian_iban);
+            output = "some iban: [iban]!";
+            remarks = vec![Remark::with_range(RemarkType::Substituted, "@iban", (11, 17))];
+        );
+
+        let invalid_russian_iban = {
+            valid_russian_iban.push('0');
+            valid_russian_iban
+        };
+
+        // Since russia has the longest iban, it won't recognize an iban longer than that.
+        assert_rule_not_applied!(
+            rule = "@iban";
+            input = &format!("some iban: {}!", invalid_russian_iban);
+        );
+
+        // Don't apply if it doesnt start with two uppercase letters.
+        assert_rule_not_applied!(
+            rule = "@iban";
+            input = "some iban: N19386011117945!";
+        );
+    }
+
+    #[test]
+    fn test_valid_iban_codes() {
+        // list taken from: `https://github.com/jschaedl/iban-validation/blob/master/tests/ValidatorTest.php`
+        // Todo: verify that it's okay to copy it.
```

Comment:
I got this list from an MIT lisenced repository, is that enough to think of it as free to use, or do i need to also verify their source?